.build-tools/setBuildVersion.ps1

```diff
@@ -6,6 +6,7 @@ param (
 )
 
 $manifestVersionAsArray = (Import-PowerShellDataFile $absolutePathToManifest).ModuleVersion -split "\."
-$updatedBuildVersion = @( $manifestVersionAsArray[0], $manifestVersionAsArray[1], $buildPatch) | Join-String -Separator '.'
+$buildPatchAsArray = $buildPatch -split "\."
+$updatedBuildVersion = @( $manifestVersionAsArray[0], $manifestVersionAsArray[1], ($buildPatchAsArray | Select-Object -Last 1) ) | Join-String -Separator '.'
 Write-Host "----- Setting Build Version to $updatedBuildVersion -----"
 Write-Host "##vso[build.updatebuildnumber]$updatedBuildVersion"
```

Comment:
If we remove these 2 lines then it doesn't matter which package builds last, it will just report the build counter as the number, which I think is fine.

Reply:
Updated the build versioning to use the build counter